jina/peapods/pea.py

```diff
@@ -7,22 +7,22 @@
 import threading
 import time
 from collections import defaultdict
-from multiprocessing.synchronize import Event
 from typing import Dict, Optional, Union, List
 
 import zmq
 
 from jina.types.message import Message
-from .zmq import send_ctrl_message, Zmqlet, ZmqStreamlet
+from .zmq import ZmqStreamlet
 from .. import __ready_msg__, __stop_msg__, Request
 from ..enums import PeaRoleType, SkipOnErrorType
-from ..excepts import NoExplicitMessage, ExecutorFailToLoad, MemoryOverHighWatermark, DriverError, PeaFailToStart, \
+from ..excepts import RequestLoopEnd, NoExplicitMessage, ExecutorFailToLoad, MemoryOverHighWatermark, DriverError, \
     ChainedPodException
 from ..executors import BaseExecutor
-from ..helper import is_valid_local_config_source, typename
+from ..helper import is_valid_local_config_source
 from ..logging import JinaLogger
 from ..logging.profile import used_memory, TimeDict
 from ..proto import jina_pb2
```

Comment:
So at this level we are still using the `jina_pb2`?

Reply:
At this level is still a little leftove (just checking a status code, I guess we could easily refactor that too), it should still be lower in the code.